go.mod

```diff
@@ -2,10 +2,9 @@ module github.com/red-hat-storage/ocs-client-operator
 
 go 1.22.0
 
-toolchain go1.22.3
-
 replace (
 	github.com/portworx/sched-ops => github.com/portworx/sched-ops v0.20.4-openstorage-rc3 // required by Rook v1.12
+	github.com/red-hat-storage/ocs-client-operator/api => ./api
```

Comment:
Should we move this to a separate replace block?

Reply:
doesn't make any diff, nack.